opengemini-client-okhttp/src/main/java/io/opengemini/client/okhttp/OkHttpSslContextFactory.java

```diff
@@ -28,7 +29,12 @@ static OkHttpSslContext createOkHttpSslContext(TlsConfig tlsConfig) {
                     tlsConfig.tlsVerifyDisabled);
             SSLContext sslContext = SSLContext.getInstance("TLSv1.3");
             sslContext.init(keyManagers, trustManagers, new SecureRandom());
-            X509TrustManager x509TrustManager = (X509TrustManager) trustManagers[0];
+            X509TrustManager x509TrustManager;
+            if (trustManagers[0] instanceof X509TrustManager) {
+                x509TrustManager = (X509TrustManager) trustManagers[0];
+            } else {
+                throw new OpenGeminiException("The trust manager is not an instance of X509TrustManager");
```

Comment:
I prefer to trhow IllegalStateException here.

Reply:
We will roll back this piece of code.